core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiServiceV2.scala

```diff
@@ -53,6 +53,22 @@ trait BillingApiServiceV2 extends UserInfoDirectives {
             }
           }
         } ~
+        pathPrefix("billingAccount") {
+          pathEnd {
+            put {
+              entity(as[UpdateRawlsBillingAccountRequest]) { updateProjectRequest =>
+                complete{
+                  userServiceConstructor(userInfo).updateBillingProjectBillingAccount(RawlsBillingProjectName(projectId), updateProjectRequest).map(_ => StatusCodes.OK)
+                }
+              }
+            } ~
+              delete {
+                complete {
+                  userServiceConstructor(userInfo).deleteBillingAccount(RawlsBillingProjectName(projectId)).map(_ => StatusCodes.OK)
```

Comment:
I originally had these as `No Content` but it looks like `v2_workspaces` returns `OK` with a body.

Any objections to returning `OK` but with no body for now?

Reply:
That seems fine to me, for the time being. I generally prefer returning the updated object to returning an empty body, because it gives clients the ability to base their updates on the resulting object, rather than guessing or making a fresh api call.

Reply:
Sorry forgot to respond to this. The reason that I first chose to return nothing here is that the format of what I would be able to return here would be quite different than what the endpoint will return once v2_workspaces is merged. I could get close, but it would require bringing in a bunch of other code from v2_workspaces.

My thinking is that it is better to go from no response body to _some_ response body, rather than going from some response body that sets expectations for field names, values, etc and then switching to something else.

In general, I totally agree with you about trying to return the updated object when applicable.

Reply:
I think I'm in agreement with you Matt.  Better to return nothing for now and then add to it.  Technically, we might consider that an API version bump, but I think it's ok here.  If we actually change the contents of what is returned, then I think we'd need to version the API again with PPW, and I don't think anybody wants to do that.